marissahuysentruyt/Pokemon-v2
# PKM-2: Feat: Landing page 🎉

Description
========
Pokemon Clash has finally gotten further into development! The landing page has been created. 

Validation
=========

To Validate
----------
1. Make sure all PR Checks have passed (GitHub Actions, CircleCI, Code Climate, etc).
1. Pull down all related branches.
1. Confirm all tests pass.
1. Make sure you can filter characters by clicking on the radio buttons.

Browser Testing
--------------
Gold Level Support

In these browsers, behavior & design closely match original specifications. A user is able to access all content and functionality, including the usability of required assistive devices, such as keyboard and screenreader.

macOS

* [ ] Safari (last 2 major versions)
* [ ] Chrome (last 6 months)
* [ ] Firefox (last 6 months)

Windows

* [ ] Chrome (last 6 months)
* [ ] Firefox (last 6 months)

Mobile

* [ ] Safari (last 2 major versions)
* [ ] Android 7+
* [ ] Samsung Browser (last year)
* [ ] Chrome (last 6 months)
* [ ] Firefox (last 6 months)
